packages/cli/README.md

```diff
@@ -25,3 +25,6 @@ https://www.tracerbench.com/docs/guide/stats-primer
 ### TracerBench Stats Class Module
 https://www.npmjs.com/package/@tracerbench/stats
 https://github.com/TracerBench/tracerbench/tree/master/packages/stats
+
+### RFC List
```

Comment:
Thanks for adding this! I think this will go a long way toward making sure this feature is known by consumers